docs/source/tracking.rst

```diff
@@ -504,6 +504,12 @@ See `Set up AWS Credentials and Region for Development <https://docs.aws.amazon.
   is a path inside the file store. Typically this is not an appropriate location, as the client and
   server probably refer to different physical locations (that is, the same path on different disks).
 
+Deletion Behavior
+~~~~~~~~~~~~~~~~~
+In order to allow MLflow Runs to be restored, Run metadata and artifacts are not automatically removed
+from the backend store or artifact store when a Run is deleted. The :ref:`mlflow gc <cli>` CLI is provided
```

Comment:
```suggestion
from the backend store or artifact store when a Run is deleted. The :ref:`mlflow gc <backend_store_uri>` CLI is provided
```

Reply:
@t-henri This is RST syntax; the bracketed code creates a link to the MLflow CLI docs page; `backend_store_uri` is not a valid link.